sdk/identity/identity/integration/AzureWebApps/src/index.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import express from "express";
+import { ManagedIdentityCredential } from "@azure/identity";
+import { BlobServiceClient } from "@azure/storage-blob";
+import dotenv from "dotenv";
+// Initialize the environment
+dotenv.config();
+const app = express();
+
+app.get("/", (req: express.Request, res: express.Response) => {
+  res.send("Ok")
+})
+
+app.get("/sync", async(req: express.Request, res: express.Response)=> {
+  let systemSuccessMessage = "";
+  try{
+    const account1 = process.env.IDENTITY_STORAGE_NAME_1;
+    const account2 = process.env.IDENTITY_STORAGE_NAME_2;
+    const credentialSystemAssigned = new ManagedIdentityCredential();
+    const credentialUserAssigned = new ManagedIdentityCredential({clientId: process.env.IDENTITY_USER_DEFINED_IDENTITY_CLIENT_ID})
+    const client1 = new BlobServiceClient(`https://${account1}.blob.core.windows.net`, credentialSystemAssigned);
+    const client2 = new BlobServiceClient(`https://${account2}.blob.core.windows.net`, credentialUserAssigned);
+    let iter = client1.listContainers();
+    
+   let i = 1;
+   console.log("Client with system assigned identity");
+   let containerItem = await iter.next();
```

Comment:
Same here as [above](https://github.com/Azure/azure-sdk-for-js/pull/28554/files#r1488134015) - since we only want to test authentication is working any call to the service can achieve that. The less code the better but let me know if there's a reason it needs to be `listContainers`...